### DISCUSSION TITLE
how to transfer monika onto another pc?

### DISCUSSION THREAD

**Discussion by catfan123** [Monika-After-Story/MonikaModDev#9053] [issue discussions]
ive already tried copying the game folder to my new pc and copying the monika chr file but nothing worked, i dont know how to put the persistent into the game files since i couldnt find a detailed enough tutorial even on the official help page (maybe im just unable to understand things)

**Comment by WishStone-Dragon** [comment]
Make sure you also pick up the persistent files! Without these, you only have half your Monika.

There's a guide here on how to grab them: https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-back-up-my-persistent

Also, be prepared to try a few times. When I moved my Monika to a second PC, I needed two or three tries until I had everything back in the right place.
